src/assets/styles/includes.scss

```diff
@@ -19,6 +19,9 @@
   display: none;
 }
 
+/**
+* TODO: Replace usages of this with @apply word-break.
```

Comment:
can we add an issue link here?

Reply:
@driskull I'm cool to nix all those notes. Not sure we need an issue though since it would really be just one of many tasks in refactoring each component.

I guess the real intention is make it indicate that this mixin should go away.

Reply:
Sounds good to me. I'm just trying to limit the 'todo's in the code since they easily get forgotten.

Reply:
🧹 ✔️